### DISCUSSION TITLE
Add vite and other nodejs apps to Herd

### DISCUSSION THREAD

**Discussion by isimmons** [beyondcode/herd-community#632] [Ideas]
Since I do other things besides Laravel and Herd has to take over control of everything else, node, nvm, mysql, nginx in order to work so well for Laravel and other PHP apps, It would be great if I could also move my React, Vue, Next, Nuxt sites into the herd.

I tried creating a Vite React app inside the Herd directory. It automatically added the correct entry to my hosts file. But I get a Herd 404 page. I assume this is because there is nothing linking my 'dist' directory. Also no node server running for SSR apps but it seems like static SPA's would be easy and a great start towards making this possible.

Herd is great and I'm loving it for Laravel. But I hope in the future we can use it for managing all things.

**Comment by delixyr** [comment]
+1
